codespeed/commits/github.py

```diff
@@ -22,6 +22,7 @@
 import isodate
 from django.conf import settings
 from django.core.cache import cache
+from django.conf import settings
```

Comment:
The build fails because of this duplicate `settings` import

Reply:
Yeah, I pushed a lint fix and synced this branch with latest master.